src/reduce.test.js

```diff
@@ -11,9 +11,57 @@ describe('reduce', () => {
     delete Array.prototype.reduce2;
   });
 
-  it('should ', () => {
+  it('should be a function', () => {
+    expect([].reduce2).toBeInstanceOf(Function);
+  });
+
+  it('should throw an error if cb is not a func or not passed', () => {
+    // eslint-disable-next-line no-undef
```

Comment:
The use of `_` as a placeholder for an undefined variable will cause a reference error. Replace `_` with `undefined` to correctly test the scenario where no callback function is provided.

Reply:
Done